enterprise/templates/enterprise/enterprise_program_enrollment_page.html

```diff
@@ -0,0 +1,172 @@
+{% extends 'enterprise/base.html' %}
```

Comment:
It's copy and pasted from the other template files because they will be sharing format a lot -- ignore for now.